mariadb-connector/src/main/java/life/qbic/projectmanagement/persistence/ProjectJpaRepository.java

```diff
@@ -40,6 +48,15 @@ public void add(Project project) {
     projectRepo.save(project);
   }
 
+  @Override
+  public List<ProjectPreview> getAllPreviews(int offset, int limit) {
+    List<ProjectPreview> projects = new ArrayList<>();
+    //todo how to handle projects with null values for e.g proejct objective?
```

Comment:
this is meant to stay in there? (just askin')

Reply:
nice catch
I will remove it